src/pkg/projects/Microsoft.NETCore.App/src/Microsoft.NETCore.App.depproj

```diff
@@ -152,6 +146,13 @@
       <_sourcePath Include="@(_sourcePathCandidate)" Condition="Exists('%(Identity)')" />
     </ItemGroup>
 
+    <!-- Add versions file with the hashes of the repos we consume -->
+    <ItemGroup>
+      <FilesToPackage Include="$(IntermediateOutputPath)\Microsoft.NETCore.App.versions.txt">
+        <TargetPath></TargetPath>
```

Comment:
How does this fix it? Is there something that is changing empty TargetPaths to something else?